mdx-guide.md

```diff
@@ -0,0 +1,346 @@
+- `[Bit ordering in the Qiskit SDK](/guides/bit-ordering)`
+
+External URLs should use the entire URL, such as `[GitHub](https://github.com)`.
+
+## Images
+
+Images are stored in the `public/images` folder. You should use subfolders to organize the files. For example, images for `guides/my-file.mdx` should be stored like `public/images/guides/my-file/img1.png`.
+
+To use the image:
+
+```markdown
+![Alt text for the image](/images/guides/your-file/your_image.png)
+```
+
+To add an inline images:
+
+```markdown
+Inline ![Alt text for the image](/images/guides/your-file/your_image.png) image
+```
+
+To include a caption:
+
+```markdown
+![Alt text for the image](/images/guides/your-file/your_image.png "Image caption")
+```
+
+You can include a version of the image to be with the dark theme. You only need to create an image with the same name ending in `@dark`. So for example, if you have a `sampler.png` image, the dark version would be `sampler@dark.png`. This is important for images that have a white background.
+
+## Videos
+
+Videos are stored in the `public/videos` folder. You should use subfolders to organize the files. For example, images for `guides/my-file.mdx` should be stored like `public/videos/guides/my-file/video1.mp4`.
+
+To add a video:
+
+```markdown
+<video title="Write a description of the video here as 'alt text' for accessibility." className="max-w-auto h-auto" controls>
+    <source src="/videos/guides/sessions/demo.mp4" type="video/mp4"></source>
+</video>
+```
+
+## Math
+
+We use [LaTeX](https://www.latex-project.org) to write math, which gets rendered by the library [KaTeX](https://katex.org).
+
+Inline math expressions should start with `$` and end with `$`, e.g. `$\frac{123}{2}$`.
+
+Multi-line expressions should start with `$$` and end with `$$`:
+
+```markdown
+$$
+L = \frac{123}{2} \rho v^2 S C_1s
+$$
+```
+
+## Tables
+
+Tables are supported: https://www.markdownguide.org/extended-syntax/.
+
+Warning: do not use `|` inside LaTeX/math expressions. Markdown will incorrectly interpret `|` as the divider between cells. Instead, use `\vert`.
+
+## Comments
+
+Example comment: `{/* Comes from https://qiskit.org/documentation/partners/qiskit_ibm_runtime/getting_started.html */}`
+
+## Collapsible sections
+
+For content that you don't want to show by default, use a collapsible section. The user will need to expand the section to read its contents. Refer to GitHub's guide on [`<details>` and `<summary>`](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections).
+
+## Footnotes
+
+```
+Footnote 1 link[^first].
+
+Footnote 2 link[^second].
+
+Duplicated footnote reference[^second].
+
+[^first]: Footnote **can have markup**
+
+    and multiple paragraphs.
+
+[^second]: Second footnote text.
+```
+
+## Trademark symbols
+
+Refer to [the style guide](./style-guide.md#proper-marking-and-attribution) for expectations on marking and attribution.
```

Comment:
The sentence is now. I moved the Proper marking section to the style guide, so link to it here.